scripts-available/CDB_Overviews.sql

```diff
@@ -1,3 +1,29 @@
+-- Maximum zoom level for which overviews may be created
+CREATE OR REPLACE FUNCTION _CDB_MaxOverviewLevel()
```

Comment:
By using bigints we could raise the maximum overview level from 23 to 29 or 30 (it'd be pointless to support higher levels because using such levels would cause problems elsewhere).
I hadn't think of that approach, basically because I think having up to 23 overview levels should be enough for most cases, but I can do a quick check to see if it a) is easy to implement b) doesn't have a performance impact (the feature density query is already quite slow).